### PR TITLE
Add 404 page

### PR DESCRIPTION
**Pull request by Arthelon**
Addresses #59 

Loosely based on the 404 page from https://skygear.io

### CODE REVIEW

**Review by kiootic — CHANGES_REQUESTED** [review]
We'll need an explicit 404 page to make it export a static HTML (e.g 404.html), so that we can configure GitHub Pages/S3 to return this page when not found.

File: src/routes/_error.svelte [hunk 1]
```diff
@@ -1,20 +1,22 @@
 <script>
-  import NotFound from "../components/NotFound.svelte";
+  import { onMount } from "svelte"; // eslint-disable-line import/no-extraneous-dependencies
+  import { goto } from "@sapper/app"; // eslint-disable-line import/no-extraneous-dependencies
 
   export let status;
   export let error;
 
+  onMount(() => {
```
Comment (kiootic) [inline src/routes/_error.svelte]: We don't need this, user can stay in the original path.

Comment (Arthelon) [reply src/routes/_error.svelte]: Ok, I'll just revert to what it was before and display the NotFound component if the error is a 404

Comment (kiootic) [reply src/routes/_error.svelte]: Also, please add a descriptive page title like 'Page Not Found'.

**Review by kiootic — COMMENTED** [review]
It seems there is still no exported 404 HTML file. Can you look into it?

**Comment by Arthelon** [comment]
Should be fixed. Outputs to `404/index.html`

**Review by kiootic — APPROVED** [review]

**Pull request merged**